server/sonar-server/src/main/java/org/sonar/server/computation/issue/IssueLifecycle.java

```diff
@@ -70,6 +76,7 @@ public void mergeExistingOpenIssue(DefaultIssue raw, DefaultIssue base) {
     raw.setAssignee(base.assignee());
     raw.setAuthorLogin(base.authorLogin());
     raw.setTags(base.tags());
+    raw.setAttributes(base.attributes());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)